multinet/api.py

```diff
@@ -24,7 +27,31 @@
 @swag_from("swagger/workspaces.yaml")
 def get_workspaces() -> Any:
     """Retrieve list of workspaces."""
-    return util.stream(db.get_workspaces())
+    user = session_user()
+    current_app.logger.info(user)
```

Comment:
Is this left over from debugging?

Reply:
Yup :)

See a9e328c4e2404460df452d0107acd500db50acac.